model/project_ref.go

```diff
@@ -316,6 +321,20 @@ const (
 	maxBatchTime             = 153722867 // math.MaxInt64 / 60 / 1_000_000_000
 )
 
+type ProjectRefSection string
+
+const (
+	ProjectRefGeneralSection        = "general"
+	ProjectRefAccessSection         = "access"
+	ProjectRefVariablesSection      = "variables"
+	ProjectRefGithubAndCQSection    = "github_and_commit_queue"
+	ProjectRefNotificationsSection  = "notifications"
+	ProjectRefAliasSection          = "alias"
```

Comment:
The section in the UI is specifically only for patch aliases. Would it be possible to change this to `PatchAliasSection  = "patch_alias"`? I think that will make things clearer since we have other kinds of aliases sprinkled throughout other pages.

Reply:
sure thing